Dockerfile

```diff
@@ -1,53 +1,22 @@
-FROM ubuntu:16.04 AS builder
-ARG NODE_VERSION="v13.11.0"
-WORKDIR /app
-COPY . /app
-RUN apt-get update && apt-get install -y \
-	build-essential \
-	curl \
-	git \
-	python3 \
-	&& rm -rf /var/lib/apt/lists/*
-WORKDIR /
-RUN curl -s -O "https://nodejs.org/download/release/${NODE_VERSION}/node-${NODE_VERSION}-linux-x64.tar.gz"
-RUN tar xzf "node-${NODE_VERSION}-linux-x64.tar.gz"
-ENV PATH="/node-${NODE_VERSION}-linux-x64/bin:${PATH}"
+FROM node:13.10
+WORKDIR /usr/src/broker
+COPY . .
+
 RUN node --version
 RUN npm --version
-RUN useradd -ms /bin/bash node
-USER node
-WORKDIR /home/node
-COPY ./ ./
-RUN npm ci
-
-FROM ubuntu:16.04
-ARG NODE_VERSION="v13.11.0"
-WORKDIR /app
-COPY --from=builder /app/ .
-RUN apt-get update && apt-get install -y \
-	curl \
-	&& rm -rf /var/lib/apt/lists/*
-
-WORKDIR /
-ENV PATH="/node-$NODE_VERSION-linux-x64/bin:${PATH}"
-COPY --from=builder --chown=root:root /node-$NODE_VERSION-linux-x64/ /node-$NODE_VERSION-linux-x64/
-
-RUN useradd -ms /bin/bash node
-USER node
-WORKDIR /home/node
-COPY --from=builder --chown=node:node /home/node/ ./
+RUN npm ci --only=production
 
 # Make ports available to the world outside this container
-EXPOSE 30315
+EXPOSE 30316
 # WebSocket
-EXPOSE 8890
+EXPOSE 8790
 # HTTP
-EXPOSE 8891
+EXPOSE 8791
 # MQTT
-EXPOSE 9000
+EXPOSE 9100
 
-ENV DEBUG=streamr:logic:*
-ENV CONFIG_FILE configs/docker-1.env.json
+ENV export DEBUG=streamr:logic:*
+ENV CONFIG_FILE configs/docker-2.env.json
```

Comment:
I'd keep maybe 'docker-1.env.json' as default for predictability?

Reply:
yep